strapi/strapi
# Only support better-sqlite3 package for sqlite

### What does it do?

Adds a migration script to support only better-sqlite3 as a sqlite package

### Why is it needed?

To make all the Strapi projects compatible with V5

### How to test it?

Execute the 2 provided commands with the upgrade tool in a package.json with a vscode/sqlite3, sqlite3, or sqlite-legacy packages and check that they are replaced with better-sqlite3. Also, check your config/database.js file and see that if you had any other client set in the client options, now you should have always sqlite.

### Related issue(s)/PR(s)

Let us know if this is related to any issue/pull request
